tools/mFold.md

```diff
@@ -0,0 +1,131 @@
+# mFold
+
+written by: Kevin Yuan
+
+[{estimated time} minute] This tool is for tutorial 11. mFold is one of the earliest tools for computational molecular biology, operating since 1995. It is primarily used for prediction of nucleic acid folding, which can be valuable in evaluating and comparing the structure of viruses.
+
+**Tutorial Objective**: We will use mFold to generate several possible structures of a truncated narnavirus.
+
+## Input / Prerequisites
+
+-   [Tool Weblink](http://www.unafold.org//)
+
+-   Web browser
+
+-   A DNA or RNA sequence \< 2400 nucleic acids
+
+-   Example Data: AGACGGATGTGATAGATAAACTATCACTAGCTGATTATGGCACTTTAAAGCGCCTAATCA
+
+    ACATTGTTGAGGGGTTTAAGGATAACTTTATGTTTTCCCAACCCGAGAGATTCTTGACTT
+
+    TGATCAAGAGTCCTCAATACATCCAGTTTATGCGTTGGGTCATTAGTACCGAAACTCATA
+
+    AACCTGGGCTTGCCATAGACCAATGGAAAGAATTTATAAATCTTATCCGTTGGCGGGCAT
+
+    TACAGTCCCAGACGGATGAACCGCCCATCCCAGTGAACTTCCCCGGCGTAGACCGGAGGA
+
+    GTTCAGCTGGAAAAATTAGGGTCTCGCCCTTTTGGCAGCGTCTATGTCCGCTCTTGGGTG
+
+    AGATCTGGGATGGTCTTAAAACAAAAGACCAAGGAACGAGGTTTATGCACTTCTGTTCCT
+
+    CTCGAAACCTTCCGGCAGCTCCTCAATGGAAGCGCGAGAAGGCGGAGTTAGAACACTATA
+
+    AAGTGTTAACTACTCCGGGACCAAATCTAGAGGCCCGTAACAAGTGGCTATATTATGCCG
+
+    CATTAGATTACGGGCGGTCCTTAATAAAACCAAATGATCGAATGACCGAACGTCAATTCG
+
+    CTCATTTGTCTCTGACAAACTCTGCAAGTATCCTATCACCTGTGCGTGATGGAGGCAGAG
+
+    GGACAGAGATAGCCGCTCGTTATAGAGAGTGGGCTATGAAACCAGCCACGGAAACTATTG
+
+    ATGAGTTAACGTGGTTTGGAATGAGATACTGGAAAGTCGAAGGCATTCCAGTATGGAGAA
+
+    CCATGTGCCGACCGCATATGGCCCCAGACGGAATACCAGGAGACTCAGGACTCCTGGAGA
+
+## Output
+
+The output of mFold is a set of generated models predicting nucleic acid folding and their respective thermodynamic properties which can be exported in a variety of file types including pdf, png, and jpg.
+
+### 1. Navigate to [the UNAFold Web Server where mFold is also hosted on](http://www.unafold.org/)
+
+-   There is a dropdown in the header of the webpage called 'mFold', where you can then select 'Applications' and subsequently, your desired program ('DNA Folding Form' or 'RNA Folding Form')
+
+    ![](images/mfold header.PNG)
```

Comment:
Please add a link title in all places in the document describing the figure in a few words.